source/coding/git.rst

```diff
@@ -1,153 +1,64 @@
 Git
 ===
 
-What are git/Github/Bitbucket?
-------------------------------
+.. role:: bash(code)
+   :language: bash
 
-Let's say you want to save your work on internet to make sure that everything is safe if your computer breaks. Let's also say you want to share your work with other people. One answer that often comes to mind is to use Dropbox. That's good, but in the special case where your work is code (in any programming language), it is not the best solution.
+If you do not know what is Git, you should visit the page (CROSSREF when done) to learn about the basic concepts and commands of Git. This page is specifically to learn how we use Git in the lab, what we expect and how you can contribute.
 
-Ex: you're trying to develop a feature on, ex, scilpy, and your collegue is developping another feature on his own computer. 1) You don't want to save it constantly, dropbox-style. You only want to add your code once you're sure it has been correctly tested. 2) If the two of you want to add your features, we need a way to deal with conflicts if you have changed the same files. Git does that. Github and bitbuckets are two websites offering git as service in a user-friendly way.
+For users
+---------
 
-You can find many good tutorials on internet, such as `this one <https://git-scm.com/doc>`_ on the official website. Or check `this very nice introduction <https://docs.google.com/presentation/d/1z0gsgM2Of3TIBSmJUVNPJJR6UkWZEEE_OBVYixOT8iA/edit#slide=id.g6b1c9a75fb_2_789.>`_ by Alex from our lab.
+*If you are only a user and you don't plan on developing the code:*
 
-How we use git in the lab
--------------------------
+You can download a repository; we say to **clone** a repository. Downloading vs cloning: when you download, it takes the code as it is then, wraps it in a zip file and downloads it all. But then, you have nothing telling you where you downloaded it from and you can't update the code. Cloning keeps the link with git and allows you to update. You will need to open a terminal and use the :bash:`git clone` command. You can find the address by clicking on Github's green button *clone repository*.
 
-*If you are only a user and your don't plan on developping the code:*
+Once cloned we recommend either using a tag (a release) with the :bash:`git checkout` to ensure a stable version of the code. We expect users to remember which repository the cloned, to put the code in a appropriate folder on their computer (and know where it is), approximately when they cloned it and to remember that Scilpy constantly evolve.
```

Comment:
I'm not sure the first sentence is easy for someone not used to it. Maybe:

Once cloned we recommend you to ensure you work with a stable version of the code. Often, programmers will separate their most recent (less tested) code from more stable versions by adding tags (releases) to the various versions. To use a specific version / tag / release, use  :bash:`git checkout VERSION`.